.nuke/build.schema.json

```diff
@@ -98,6 +98,8 @@
             "type": "string",
             "enum": [
               "BuildAll",
+              "BuildAndLoadLocallyKubernetesTentacleContainerImage",
+              "BuildAndPushKubernetesTentacleContainerImage",
```

Comment:
This keeps getting generated when I do a nuke build. I'm just checking it in so it's consistent with the others.